cg/services/illumina/backup/backup_service.py

```diff
@@ -145,6 +151,20 @@ def _process_run(
 
         return get_elapsed_time(start_time=start_time)
 
+    def _get_sequencing_run_output_dir(self, archived_run: Path, is_current: bool) -> Path:
+        """Return the path to the sequencing run based on the archived run path."""
+        if is_current:
+            run_full_name: str = archived_run.parent.name
+        else:
+            run_full_name = archived_run.stem
```

Comment:
Is this really going to be the same as archived_run.name.split(".")[0]? Aren't the archives named rundir.tar.gz.gpg? Would this not return rundir.tar.gz, which is not the sequencing_run_output_dir we want?

Reply:
fixed